### PR TITLE
[test ui] Encode URLs in free response

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: apps/src/code-studio/reporting.js [hunk 1]
```diff
@@ -48,7 +48,8 @@ reporting.sendReport = function (report) {
       queryItems.push(key + '=' + report[key]);
     }
   }
-  var queryString = queryItems.join('&');
+
+  var queryString = encodeURI(queryItems.join('&'));
```
Comment (joshlory) [inline apps/src/code-studio/reporting.js]: I think you want to encode each item in `queryItems` before joining?  I believe we should use `encodeURIComponent` instead.

**Comment by joshlory** [comment]
A heads up, the UI test you modified isn't run by CircleCI:

```
UI tests for ChromeLatestWin7_free_response_submittable skipped (0:15 minutes)
We didn't actually run any tests, did you mean to do this?
```

**Comment by mehalshah** [comment]
Encoded the items individually, and with encodeURIComponent

Ran the UI test locally (it passed).

File: apps/src/code-studio/reporting.js [hunk 2]
```diff
@@ -45,11 +45,13 @@ reporting.sendReport = function (report) {
   var queryItems = [];
   for (var key in report) {
     if (report.hasOwnProperty(key) && key !== 'onComplete') {
-      queryItems.push(key + '=' + report[key]);
+      queryItems.push(key + '=' + encodeURIComponent(report[key]));
     }
   }
   var queryString = queryItems.join('&');
 
+  console.log(queryString);
```
Comment (joshlory) [inline apps/src/code-studio/reporting.js]: Remove?

Comment (mehalshah) [inline apps/src/code-studio/reporting.js]: Whoops - fixed

**Comment by joshlory** [comment]
LGTM.  Long term it would be good to figure out this issue so we're not building the form data by hand:

```
// jQuery can do this implicitly, but when url-encoding it, jQuery calls a method that
// shows the result dialog immediately
```

**Pull request merged**